src/Textfit.js

```diff
@@ -23,7 +23,8 @@ export default class TextFit extends React.Component {
     static propTypes = {
         children: PropTypes.oneOfType([
             PropTypes.string,
-            PropTypes.func
+            PropTypes.func,
+            PropTypes.array
```

Comment:
Now that we support all variations of children, instead of listing the array as an option, it will probably work if we simplify it:

```js
children: PropTypes.node,
```